src/types/ITypes.ts

```diff
@@ -1 +1,9 @@
-export type IVariant = 'primary' | 'secondary' | 'success' | 'danger' | 'link' | 'dark' | 'plain'
+export type IVariant = 'default' | 'primary' | 'warning' | 'danger' | 'success'
```

Comment:
Estes types tem o prefixo de interface (`I`), mas são `types`. Está correto assim?

Reply:
o de variantes já existia, então pensei em manter o padrão. acha que seria uma boa mudar?

Reply:
Pior que eu to bem afastado do projeto do DS ultimamente. Acho que o Simão vai saber melhor. O que tu acha @simaomeyrer?

Reply:
@simaomeyrerdooca @antonydosreis feito